ang/civicase/activity/factories/get-activity-feed-url.factory.js

```diff
@@ -0,0 +1,40 @@
+(function (angular, $, _, CRM) {
+  var module = angular.module('civicase');
+
+  module.factory('getActivityFeedUrl', function ($route, $location, $sce) {
+    return function (caseId, category, statusType, status, id) {
+      caseId = parseInt(caseId, 10);
```

Comment:
I know we have moved this code from somewhere, but can we segregate the variables together in a way that new vars are declared/defined before existing vars
```
var af = {};
var currentPath = $location.path();
caseId = parseInt(caseId, 10);
```
And also add a new line after vars declarations and ending if statements.

Reply:
I would rather not fix any exiting code, but as the number of comments are low, I am working on the fixes :)